README.md

```diff
@@ -73,4 +75,99 @@ Django-major-event-log was developed at the UNT Libraries.
 Contributors:
 
 * [Gio Gottardi](https://github.com/somexpert)
+* [Damon Kelley](https://github.com/damonkelley)
+
+
+Developing/Testing
```

Comment:
I would have Developing and Running the Tests as separate headings at this level.